app.js

```diff
@@ -199,6 +238,7 @@ app.get('/terms-of-service', (req, res) => res.render('tos'))
 // API routes
 app.use('', apiRoutes)
 app.use('', serviceRoutes)
+app.use('/', authRouter)
```

Comment:
pretty temporary, though I'd also like to learn more about the way our current implementation of routes differs from many express examples I've seen. not a problem to conform to what we have already, though.

Reply:
Can you say more about this?

Reply:
The general rule of thumb is that our implementation may differ from current examples because the conventions change over time. We are not necessarily following every convention change closely, and updating them to match. This is especially true when old patterns are not explicitly deprecated, or if we don't have good reasons (beyond style) to change.